TheModernIlluminati/TheModernIlluminati/Pages/Country.cshtml

```diff
@@ -8,7 +8,7 @@
 
     <div class="jumbotron text-center">
         <h2 class = "text-dark" >List of Countries with Nobel Laureates </h2>
-        <p class="p-3 mb-2 bg-secondary text-white">Some text here!</p>
+        <p class="p-3 mb-2 bg-secondary text-white">Please find the country codes below!</p>
```

Comment:
Replaced with more descriptive and relatable text in according to the page details.
https://docs.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-5.0&tabs=visual-studio